src/main/kotlin/app/batch/Validator.kt

```diff
@@ -54,6 +61,15 @@ class Validator {
         return null
     }
 
+    fun replaceElementValueWithKeyValuePair(objectWithFieldIn: JsonObject, keyToReplace: String, newKey: String, value: String): Pair<JsonObject, String> {
+        var objectWithChangedField = objectWithFieldIn
```

Comment:
Does this create a deep copy of the object? If not these would be 2 pointers to the same object and if your plan was to not edit the `objectWithFieldIn` then this is not achieving that?

Reply:
Python copies the item in memory by default, so not a pointer